hibernate-reactive-core/src/test/java/org/hibernate/reactive/types/JavaTypesArrayTest.java

```diff
@@ -265,15 +343,57 @@ public void testBigIntegerArrayType(VertxTestContext context) {
 	}
 
 	@Test
-	public void testBigDecimalArrayType(VertxTestContext context) {
+	@EnabledFor({POSTGRESQL, COCKROACHDB})
+	public void testBigDecimalArrayTypeNoAnnotation(VertxTestContext context) {
 		Basic basic = new Basic();
 		BigDecimal[] dataArray = {BigDecimal.valueOf( 123384967L ), BigDecimal.ZERO};
-		basic.bigDecimalArray = dataArray;
+		basic.bigDecimalArrayNoAnnotations = dataArray;
 
 		testField( context, basic, found -> {
-			assertEquals( dataArray.length, found.bigDecimalArray.length );
-			assertEquals( dataArray[0].compareTo( found.bigDecimalArray[0] ), 0 );
-			assertEquals( dataArray[1].compareTo( found.bigDecimalArray[1] ), 0 );
+			assertEquals( dataArray.length, found.bigDecimalArrayNoAnnotations.length );
+			assertEquals( dataArray[0].compareTo( found.bigDecimalArrayNoAnnotations[0] ), 0 );
+			assertEquals( dataArray[1].compareTo( found.bigDecimalArrayNoAnnotations[1] ), 0 );
+		} );
+	}
+
+	@Test
+	@EnabledFor({POSTGRESQL, COCKROACHDB})
+	public void testBigDecimalArrayTypeWithColumnAnnotation(VertxTestContext context) {
+		Basic basic = new Basic();
+		BigDecimal[] dataArray = {BigDecimal.valueOf( 123384967L ), BigDecimal.ZERO};
+		basic.bigDecimalArrayWithColumnAnnotation = dataArray;
+
+		testField( context, basic, found -> {
+			assertEquals( dataArray.length, found.bigDecimalArrayWithColumnAnnotation.length );
+			assertEquals( dataArray[0].compareTo( found.bigDecimalArrayWithColumnAnnotation[0] ), 0 );
+			assertEquals( dataArray[1].compareTo( found.bigDecimalArrayWithColumnAnnotation[1] ), 0 );
+		} );
+	}
+
+	@Test
+	@EnabledFor({POSTGRESQL, COCKROACHDB})
```

Comment:
Why is this only enabled for these databases?

Reply:
fixed and now all tests run except for ORACLE DB